index.html

```diff
@@ -79,6 +83,155 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
           </ul>
         </div>
       </section>
+      <section class="innerSection" id="portfolioSection">
+        <h2 class="sectionTitle">My Recent Works</h2>
+
+        <!-- my recent works container -->
+        <div class="sectionGrid">
+          <div class="sectionCard">
+            <img
+              class="cardImage"
+              src="asset/image/icons/imgPlaceholder.png"
+              alt="project image placeholder"
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div class="cardTags">
+              <p class="tagText">Ruby on rails</p>
+              <p class="tagText">CSS</p>
+              <p class="tagText">JavaScript</p>
+              <p class="tagText">HTML</p>
+            </div>
+            <button class="sectionBtn">See Project</button>
```

Comment:
For the 'see project' buttons kindly include the button interactions (hover, active, etc... ) from the Figma template ✔😃✔

![image](https://user-images.githubusercontent.com/87448628/164758570-2b92bd0d-b2f4-44ca-90c7-833407c9ea4d.png)